runtime/xamarin/runtime.h

```diff
@@ -109,6 +115,7 @@ struct AssemblyLocations {
 };
 
 void xamarin_initialize ();
+void xamarin_initialize_embedded (); /* STABLE */
```

Comment:
Can we make this more explicit in the source comment? Had the same question as Sebastien when I saw it. Just to help our future selves and anyone else 🙂